### PR TITLE
PWN-6305 - Off-ramp. Analytics

### PR DESCRIPTION
**Pull request by gslevinkov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-6305

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/auth/analytics/GeneralAnalytics.kt [hunk 1]
```diff
@@ -2,14 +2,24 @@ package org.p2p.wallet.auth.analytics
...
 import org.p2p.wallet.common.analytics.constants.EventNames
+import org.p2p.wallet.common.di.AppScope
+import org.p2p.wallet.sell.interactor.SellInteractor
+import kotlinx.coroutines.launch
 
-class GeneralAnalytics(private val tracker: Analytics) {
+class GeneralAnalytics(
+    private val tracker: Analytics,
+    private val sellInteractor: SellInteractor,
+    private val appScope: AppScope
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/auth/analytics/GeneralAnalytics.kt:12]: god damn, appScope in analytics, what a world we are living in.. 
any ideas on improvement?

Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/auth/analytics/GeneralAnalytics.kt:12]: let's pass parameter in every place it's called

File: app/src/main/java/org/p2p/wallet/home/analytics/BrowseAnalytics.kt [hunk 2]
```diff
@@ -14,8 +14,15 @@ import org.p2p.wallet.common.analytics.constants.EventNames.BROWSE_TOKEN_CHOSEN
...
+import org.p2p.wallet.common.di.AppScope
+import org.p2p.wallet.sell.interactor.SellInteractor
+import kotlinx.coroutines.launch
 
-class BrowseAnalytics(private val tracker: Analytics) {
+class BrowseAnalytics(
+    private val tracker: Analytics,
+    private val sellInteractor: SellInteractor,
+    private val appScope: AppScope,
+) {
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/home/analytics/BrowseAnalytics.kt:25]: same here

**Review by 1sakov — APPROVED** [review]
minor comments

**Review by Davran331122 — APPROVED** [review]

**Pull request merged**